install_config/router/customized_haproxy_router.adoc

```diff
@@ -115,21 +115,25 @@ The *define* action names the file that will contain the processed template.
 [cols="2*", options="header"]
 |===
 |Function | Meaning
-|`*endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint*` | Returns the list of valid endpoints.
```

Comment:
Is `endpointsForAlias` considered deprecated now?

Reply:
I don't know. It is not currently used.